### PR TITLE
Transferred resetTimeout to state called system

### PR DESCRIPTION
**Pull request by LiliiaShlikhtLuxoft**
Transferred resetTimeout to state called system

### CODE REVIEW

**Review by YuriiYefimenko — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -72,6 +72,8 @@ class HMIApp extends React.Component {
         this.onTouchMove = this.onTouchMove.bind(this);
         this.onTouchEnd = this.onTouchEnd.bind(this);
         this.onTouchEvent = this.onTouchEvent.bind(this);
+        //setting default value
+        store.dispatch(resetTimeout());
```
Comment (ValeriiMalkov) [inline src/index.js:76]: @LiliiaShlikhtLuxoft Seems [here](https://github.com/LuxoftSDL/generic_hmi/blob/d0f7a4f3874f095994e6955b13e6fc642af97aaa/src/js/reducers.js#L798) is a place for default initialization

**Review by ValeriiMalkov — CHANGES_REQUESTED** [review]

**Comment by ValeriiMalkov** [comment]
@LiliiaShlikhtLuxoft Why the branch prefix is `fix` ? I dont see any fix in this PR. Seems it is review PR

File: src/index.js (hunk 1 above)
Comment (LiliiaShlikhtLuxoft) [reply src/index.js:76]: @ValeriiMalkov without dispatch resetTimeout will not appear.
You can find the same logic for ptuWithModemEnabled [here](https://github.com/LuxoftSDL/generic_hmi/blob/d0f7a4f3874f095994e6955b13e6fc642af97aaa/src/index.js#L88)

Comment (ValeriiMalkov) [reply src/index.js:76]: This is not a default initilialization. This method will called after toggle checkbox [here](https://github.com/LuxoftSDL/generic_hmi/blob/d0f7a4f3874f095994e6955b13e6fc642af97aaa/src/index.js#L233)
This is not the same logic

Comment (LiliiaShlikhtLuxoft) [reply src/index.js]: @ValeriiMalkov fixed [here](https://github.com/LuxoftSDL/generic_hmi/pull/28/commits/6ec70423976c8a3efe487a723e0f22195e5241ff)

**Review by ValeriiMalkov — APPROVED** [review]

**Pull request merged**